playbooks/viaq-store.yml

```diff
@@ -113,16 +113,8 @@
         - "openshift-ansible-lookup-plugins"
         - "openshift-ansible-playbooks"
         - "openshift-ansible-roles"
-    - name: Copy inventory and vars.yml
-      copy:
-        src: "../conf/{{ item }}"
-        dest: /root/
-        mode: 0644
-      with_items:
-        - "vars.yml"
-        - "ansible-inventory-openshift"
     - name: Run viaq playbooks
-      shell: ANSIBLE_LOG_PATH=/tmp/ansible.log ansible-playbook -v -e @/root/vars.yml -i /root/ansible-inventory-openshift /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml
```

Comment:
I need to make this a variable to deploy based on parameter either origin or openshift

Reply:
Updated to a variable with a default for upstream path.